apollo-cache-interceptor/src/commonMain/kotlin/com/apollographql/apollo/interceptor/cache/CacheExecutionContext.kt

```diff
@@ -0,0 +1,17 @@
+package com.apollographql.apollo.interceptor.cache
+
+import com.apollographql.apollo.api.ApolloExperimental
+import com.apollographql.apollo.api.ExecutionContext
+import com.apollographql.apollo.api.Operation
+import com.apollographql.apollo.api.Response
+
+@ApolloExperimental
+data class CacheExecutionContext(
```

Comment:
internal?

Reply:
I used this to carry the information about whether a given response came from the cache or not so ultimately the user should have access to this. 

See `Response<D>.cacheContext()`

Reply:
My bad you are right.